scripts/qemu/riscv64-virt.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Shell script for running Phoenix-RTOS on QEMU (riscv64-virt)
+#
+# Copyright 2021 Phoenix Systems
+# Author: Lukasz Kosinski
+#
+
+CMD="qemu-system-riscv64 -machine virt"
+
+if [ "$#" -eq 0 ]; then
+	# No args (default command)
+	CMD+=" -bios ../../_boot/phoenix-riscv64-virt.osbi -serial stdio"
```

Comment:
I think it could be more intelligent, currently If I write
```bash
./scripts/qemu/riscv64-virt.sh -kernel ./_boot/kernel.bin
```
I would also need to explicitly provide `-bios` and `-serial` params.

I would change the semantic to have default values per parameter and use them if no command line override was provided.
(the same applies to other script)

Reply:
The scripts were changed to keep default configuration (so that we no longer need to add -bios and -serial params). It is possible to override default option by passing it again (e.g. -vga std -vga cirrus - the latter option will be used by QEMU).